src/custom/state/orders/utils.test.ts

```diff
@@ -22,6 +23,19 @@ const ORDER = generateOrder({ owner: '0x...', sellToken, buyToken })
 ORDER.sellAmount = '1000'
 ORDER.buyAmount = '1000'
 
+describe('orderPriceAndCurrentPriceDiff', () => {
+  const order = { ...ORDER, kind: OrderKind.SELL }
+
+  test('returns an obj with props orderPrice, currentPrice of class Price', () => {
+    const price: PriceInformation = { token: buyToken.address, amount: '1001' }
+    expect(orderPriceAndCurrentPriceDiff(order, price.amount as string)).toMatchObject({
+      orderPrice: expect.any(Price),
+      currentPrice: expect.any(Price),
+      percentageDifference: expect.any(Percent),
+    })
```

Comment:
Can you also test the contents of the returns rather than just the types?

Reply:
Done!